### DISCUSSION TITLE
sql matches stats

### DISCUSSION THREAD

**Discussion by think4k** [SmileYzn/PugMod-AMXX#54] [Ideas]
hi SmileYzn can add match stats logged to database (k/d/ hs...)
check https://imgur.com/a/Q42WVYC

**Comment by SmileYzn** [comment]
Already doing that, check PugRank.sma plugin and here: 

https://github.com/SmileYzn/PugMod/tree/master/addons/amxmodx/stats

**Reply by think4k** [reply]
i installed but only empty page idk why!!

mysql  Ver 14.14 Distrib 5.7.38
PHP 7.2.24-0ubuntu0.18.04.11

**Comment by SmileYzn** [comment]
I can't support web installs or server installs.
Make sure that you can see what is wrong or give more details, i can't prevent based only on program versions.

Ps. Also provide what steps you made to put stats work and make sure that PugRang.amxx is running in server side too.
